docs/source/api-ref/mo.rst

```diff
@@ -13,11 +13,13 @@ profiles, and policies are logical entities represented as MOs.
 
 Accessing Properties
 ---------------------
-When you create a managed object (MO), you can access properties as follows::
+When you create a managed object (MO), you can access properties as follows:
```

Comment:
The code-block statement is included in the diff.

 

From: Mike Timm [mailto:notifications@github.com] 
Sent: Friday, April 03, 2015 10:42 AM
To: datacenter/cobra
Cc: Zach Seils (seils)
Subject: Re: [cobra] docs: covert all examples in docs to python terminal output (#94)

 

In docs/source/api-ref/mo.rst <https://github.com/datacenter/cobra/pull/94#discussion_r27732606> :

> @@ -13,11 +13,13 @@ profiles, and policies are logical entities represented as MOs.
>  
>  Accessing Properties
>  ---------------------
> -When you create a managed object (MO), you can access properties as follows::
> +When you create a managed object (MO), you can access properties as follows:

We need either a double colon or a '..code-block:: python' so syntax highlighting works in the doc build.

—
Reply to this email directly or view it on GitHub <https://github.com/datacenter/cobra/pull/94/files#r27732606> .  <https://github.com/notifications/beacon/AF3lYNygCCJlO4WWc9RuFKGRHiTc2XeFks5n7p5PgaJpZM4D5z2C.gif>